pkg/server/internal_server.go

```diff
@@ -15,7 +15,7 @@ type Config struct {
 
 // RegisterFlags add internal server flags to flagset
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
-	f.StringVar(&cfg.Config.HTTPListenAddress, "internal-server.http-listen-address", "localhost", "HTTP internal server listen address.")
```

Comment:
With an empty string here, does the underlying implementation listen on everything or local or fail?

Reply:
it listens on everything which is the default behavior for our main server as well. it took me awhile to figure out why the internal server wasn't responding and this was the reason.

imo, if you turn on/configure internal server you know you are exposing a new port and at that time you will make configuration choices to secure it in whatever way makes sense for your org.

right now if you do the minimal config to start the internal server it refuses requests and it's difficult to determine why.

Reply:
Nice, I agree listen on everything unless you know better.

Reply:
`netstat -luntp` or `apk add iproute2 && ss -ltn` are friends of mine.

Reply:
i am not the networking wizard you are :)